tests/test_account.rb

```diff
@@ -46,5 +46,9 @@ def test_account_transactions
             assert_equal(response["booked"], nil)
         end
 
+        def test_account_transactions_with_date_range
+            omit('Need a test account with enough transactions to filter by date to test this')
```

Comment:
As far as I could tell, the test above this one asserts that fetching txs works by fetching `nil` transactions from the test account. If we are to test date range I think we need a test account that has enough txs for them to be filterable by date?